webapp/src/util/hubState.ts

```diff
@@ -116,7 +116,9 @@ export function connectToHub({
         webSocket.addEventListener("open", function () {
             lastHubUpdate = Date.now();
 
-            try {
+            // we need to wait a bit before sending these messages
+            // because websocket is not fully connected yet
+            setTimeout(() => {
```

Comment:
This was throwing an error saying that the socket was still in connecting state.  It's okay to push out the sending of ident and subscriptions.